tests-integration/tests/http.rs

```diff
@@ -313,6 +313,19 @@ pub async fn test_prom_http_api(store_type: StorageType) {
         .await;
     assert_eq!(res.status(), StatusCode::OK);
 
+    // labels
+    let res = client
+        .get("/api/v1/labels?match[]=up&start=0&end=100")
```

Comment:
Could you add another case that pass multiple `match[]`?

Reply:
Ok, maybe I should remove the `start` and `end`, because we have the default value.